x/fundraising/keeper/auction.go

```diff
@@ -364,21 +366,20 @@ func (k Keeper) AllocateVestingPayingCoin(ctx sdk.Context, auction types.Auction
 	return nil
 }
 
+// ReleaseRemainingSellingCoin releases the remaining selling coin to the auctioneer.
```

Comment:
This function should be called after calling `AllocateSellingCoin`, right? If so, we need to explain this somewhere of the documents.

Reply:
Something like "This function is called right after the selling coin is released"?